app/Platform/Concerns/ActsAsPlayer.php

```diff
@@ -41,7 +40,7 @@ public function rollConnectToken(): void
         do {
             $this->appToken = Str::random(30);
         } while ($this->where('appToken', $this->appToken)->exists());
-        $this->appTokenExpiry = Carbon::now()->addDays(ConnectApiController::TOKEN_EXPIRY_DAYS);
+        $this->appTokenExpiry = Carbon::now()->addDays(14);
         $this->save();
     }
```

Comment:
`rollConnectToken` is only used by `SeedsUsers` - and differs in functionality from everywhere else that generates an appToken.

https://github.com/RetroAchievements/RAWeb/blob/8e0da0982f3f4bfb71092ff95e17e9d006f6c077/app/Helpers/database/user-auth.php#L115-L116

https://github.com/RetroAchievements/RAWeb/blob/8e0da0982f3f4bfb71092ff95e17e9d006f6c077/app/Helpers/database/user-auth.php#L122-L125

There's even code to throw out the token if it's not 16 characters long.

https://github.com/RetroAchievements/RAWeb/blob/8e0da0982f3f4bfb71092ff95e17e9d006f6c077/app/Connect/Actions/LoginAction.php#L95-L98